### DISCUSSION TITLE
Extracting only semantic roof surfaces for QGIS

### DISCUSSION THREAD

**Discussion by misonishiki** [cityjson/specs#118] [Q&A]
Hi guys!

I'm using LoD2.2 data on QGIS. Goal: Isolate roof surface polygons to calculate surface area and slope (using QGIS). However, when I select only roof surfaces in QGIS, it is incomplete, as you can see (selected roof surfaces are coloured as highlighter yellow). I'm not sure why this is the case... was wondering what you folks think. 

I have two options at this point in time: 
(1) Complete roof surfaces should/must be available on cityJSON...so I would have to extract directly from the CityJSON model using cjio?...I've followed the tutorial on cjio but I must admit, this does NOT come naturally to me. 
(2) This is normal. CityJSON isn't perfect, fill in the other roof surfaces manually on QGIS

HELP HELP HELP

HELP <3

**Comment by balazsdukai** [comment]
Could you give more details on what is incomplete in QGIS? I don't see the problem that you describe.

**Reply by misonishiki** [reply]
Damn, completely 4got

![image](https://user-images.githubusercontent.com/107499148/173658751-f3a6a7fc-b221-4c85-8450-17da1dd70d7b.png)

**Reply by misonishiki** [reply]
Sorry~~

**Comment by hugoledoux** [comment]
CityJSON **is perfect** 😃, but maybe the dataset you use might not be. Is this a part of the 3dbag.nl? If yes then all surfaces should have a semantic yes.

**Comment by misonishiki** [comment]
SOLVED!
